core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -98,20 +101,15 @@ public KafkaStore(SchemaRegistryConfig config,
     this.serializer = serializer;
     this.localStore = localStore;
     this.noopKey = noopKey;
-    
-    // TODO: Do not use the commit interval until the decision on the embedded store is done
-    int commitInterval = config.getInt(SchemaRegistryConfig.KAFKASTORE_COMMIT_INTERVAL_MS_CONFIG);
+
```

Comment:
I noticed that these constants (starting line 60) are all unused
```
  private static final String CLIENT_ID = "schema-registry";
  private static final int SOCKET_BUFFER_SIZE = 4096;
  private static final long LATEST_OFFSET = -1;
  private static final int CONSUMER_ID = -1;
```

not due to your changes - but might be worth cleaning up anyway